src/main/java/htsjdk/tribble/readers/PositionalBufferedStream.java

```diff
@@ -28,7 +28,11 @@
 
 /**
  * A wrapper around an {@code InputStream} which performs it's own buffering, and keeps track of the position.
- * 
+ *
+ * TODO: This class implements Positional, which in turn extends LocationAware, which requires preservation of
```

Comment:
This could be fixed by throwing an `IllegalArgumentException` if the `InputStream` is a `BlockCompressedInputStream` as a temporary solution. It is much better than violate the contract and assume that it is working as expected...